doc/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -120,6 +120,39 @@ The ``moveit_servo/config`` folder contains two examples of converting `SpaceNav
 
 ``spacenav_cpp.launch`` launches a C++ node that does the same thing but with less latency. We do not plan to accept C++ pull requests for more controller types because there is a lot of overhead involved in supporting them.
 
+Configuring Keyboard as a Control Device
+------------------------------------------------------
+Assuming you are in ``~/ws_moveit`` folder already, ``cd`` to ``src/universal_robot`` and checkout the latest ``noetic-devel`` branch with ``git checkout noetic-devel``. After this, you should have a ``ur5_moveit_tutorials`` package under ``universal_robot`` folder. Finally:
```

Comment:
I don't see a `ur5_moveit_tutorials` package here:  https://github.com/ros-industrial/universal_robot/tree/noetic-devel

Am I looking in the wrong place?

Reply:
Yeah. The package exists on my fork. I also have a PR for that -> https://github.com/ros-industrial/universal_robot/pull/688. Unfortunately there isn't activity since I've created that PR.